lib/ansible/module_utils/facts/system/platform.py

```diff
@@ -43,6 +44,7 @@ def collect(self, module=None, collected_facts=None):
         # platform.system() can be Linux, Darwin, Java, or Windows
         platform_facts['system'] = platform.system()
         platform_facts['kernel'] = platform.release()
+        platform_facts['kernel_version'] = platform.version()
```

Comment:
https://docs.python.org/2/library/platform.html#platform.version  <= does not seem to be kernel but release version

Reply:
```platform.release()``` return the release name but the fact is named ```kernel``` so I follow this way. Kernel seems to be more user friendly that release no ?